docs/pages/versions/unversioned/sdk/imagepicker.md

```diff
@@ -192,10 +195,21 @@ Display the system UI for taking a photo with the camera. Requires `Permissions.
-  - The `uri` property is a URI to the local image or video file (usable as the source of an `Image` element, in the case of an image) and `width` and `height` specify the dimensions of the media.
-  - The `exif` field is included if the `exif` option is truthy, and is an object containing the image's EXIF data. The names of this object's properties are EXIF tags and the values are the respective EXIF values for those tags.
-  - The `base64` property is included if the `base64` option is truthy, and is a Base64-encoded string of the selected image's JPEG data. If you prepend this with `'data:image/jpeg;base64,'` to create a data URI, you can use it as the source of an `Image` element; for example: `<Image source={'data:image/jpeg;base64,' + launchCameraResult.base64} style={{width: 200, height: 200}} />`.
-  - The `duration` property is the length of the video in milliseconds.
+
+- The `uri` property is a URI to the local image or video file (usable as the source of an `Image` element, in the case of an image) and `width` and `height` specify the dimensions of the media.
+- The `exif` field is included if the `exif` option is truthy, and is an object containing the image's EXIF data. The names of this object's properties are EXIF tags and the values are the respective EXIF values for those tags.
+- The `base64` property is included if the `base64` option is truthy, and is a Base64-encoded string of the selected image's JPEG data. If you prepend this with `'data:image/jpeg;base64,'` to create a data URI, you can use it as the source of an `Image` element; for example: `<Image source={'data:image/jpeg;base64,' + launchCameraResult.base64} style={{width: 200, height: 200}} />`.
+- The `duration` property is the length of the video in milliseconds.
+
+> **Note:** make sure that you handling `MainActivity` destruction on **Android**. See [ImagePicker.addOnPendingResultListener](#imagepickeraddonpendingresultlistenerlistener).
+
+### `ImagePicker.addOnPendingResultListener(listener)`
+
+Android system sometimes kills the `MainActivity` after the `ImagePicker` finishes. When this happens, we lost the data selected from the `ImagePicker`. However, you can retrieve the lost data in this situation adding the pending result listener. It will be called after the `MainActivity` reloads. We recommend to adding this in the global scope of your JavaScript bundle.
+
+#### Arguments
+
+- **listener (_function_)** - This function is called when `ImagePicker` has pending results. It receives an object of a type similar to the result of the `ImagePicker.launchImageLibraryAsync` or `ImagePicker.launchCameraAsync` if the `ImagePicker` finished successfully. Otherwise, it receives [ImagePicker.ImagePickerErrorResult](#imagepickerimagepickererrorresult).
```

Comment:
```suggestion
- **listener (_function_)** - This function is called when `ImagePicker` holds some pending results. It receives an object of exactly same type as in `ImagePicker.launchImageLibraryAsync` or `ImagePicker.launchCameraAsync` if the `ImagePicker` finished successfully. Otherwise, it receives [ImagePicker.ImagePickerErrorResult](#imagepickerimagepickererrorresult).
```